src/kamel.ts

```diff
@@ -120,14 +128,17 @@ async function kamelInternal(command: string, devMode: boolean, namespace : stri
 	});
 }
 
-async function kamelInternalArgs(args: string[], devMode: boolean, namespace: string | undefined, foldername?: string): Promise<ChildProcess> {
+async function kamelInternalArgs(args: string[], devMode: boolean, namespace: string | undefined, operatorId: string | undefined, foldername?: string): Promise<ChildProcess> {
```

Comment:
Maybe better to use `operatorId?`like with foldername

Reply:
I tried to keep the same level of requirement than for `namespace`. But maybe better to use the ? operator for all. Long time that I have not read this code so do not remember the reasoning of not using it. Can be interesting to have a closer look effectively.